dvc/exceptions.py

```diff
@@ -178,6 +178,17 @@ def __init__(self):
         )
 
 
+class NoMetricsParsedError(DvcException):
+    def __init__(self, metrics):
+        lines = "\n".join(
+            f"{rev}: {', '.join(str(p) for p in paths)}"
+            for rev, paths in metrics.items()
```

Comment:
output might become a bit long with this I guess. I wonder would be the best to just mention that `-v` is available? Also, considering that `-v` is still needed to understand what is actually going on, I guess.

In that case we could simplify the logic a bit - boolean flag below would be enough instead of a full dict.